### PR TITLE
Add support for Mongoid 8

### PR DESCRIPTION
**Pull request by wulffeld**
Hopefully fixes https://github.com/mongoid/mongoid-grid_fs/issues/78. Also need this for carrierwave-mongoid.

### CODE REVIEW

**Comment by dblock** [comment]
Travis CI is long MIA. Can we do a GHA upgrade first in a separate PR and maybe a Mongoid 8 compat? You can copy-paste from https://github.com/mongoid/mongoid-compatibility/pull/13 for example.

@wulffeld want to give it a shot?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-### 2.4.1 (Next)
+### 2.5.0 (Next)
 
-* Your contribution here.
```
Comment (dblock) [inline CHANGELOG.md:3]: Put this line back.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-### 2.4.1 (Next)
+### 2.5.0 (Next)
```
Comment (dblock) [inline CHANGELOG.md:1]: Also need to increment version in version.rb.

**Review by dblock — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (wulffeld) [reply CHANGELOG.md:1]: Ok, maybe 2.5 is too big a jump? Should I stick with 2.4.1?

**Comment by wulffeld** [comment]
> Travis CI is long MIA. Can we do a GHA upgrade first in a separate PR or part of this? You can copy-paste from [mongoid/mongoid-compatibility#13](https://github.com/mongoid/mongoid-compatibility/pull/13) for example.

Yeah so I noticed. I think a separate PR might be best.

File: CHANGELOG.md (hunk 1 above)
Comment (wulffeld) [reply CHANGELOG.md:3]: Done.

**Comment by dblock** [comment]
@wulffeld I don't want to merge any changes without CI. Care to do the Travis -> GHA in a separate PR first?

**Pull request closed without merging**